sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -144,13 +143,14 @@ public void testAvroIOWriteAndReadASingleFile() throws Throwable {
     File outputFile = tmpFolder.newFile("output.avro");
 
     p.apply(Create.of(values))
-      .apply(AvroIO.Write.to(outputFile.getAbsolutePath())
+      .apply(AvroIO.<GenericClass>write().to(outputFile.getAbsolutePath())
           .withoutSharding()
           .withSchema(GenericClass.class));
     p.run();
 
     PCollection<GenericClass> input = p
-        .apply(AvroIO.Read.from(outputFile.getAbsolutePath()).withSchema(GenericClass.class));
+        .apply(AvroIO.<GenericClass>read().from(outputFile.getAbsolutePath())
```

Comment:
for TextIO I added a helper writeStrings method to prevent a usability regression. Are you suggesting something like that here?  AvroIO.writeGenerics()